blobstore/postgres_filesystem.go

```diff
@@ -34,11 +36,24 @@ $$ LANGUAGE plpgsql;`,
     AFTER DELETE ON files
     FOR EACH ROW EXECUTE PROCEDURE delete_file();`,
 	)
-	return &PostgresFilesystem{db: db}, m.Migrate(db)
+	// TODO(jpg) reuse pkg/postgres connection when converted
+	connConf := pgx.ConnConfig{
+		Host:     os.Getenv("PGHOST"),
+		User:     os.Getenv("PGUSER"),
+		Password: os.Getenv("PGPASSWORD"),
+		Database: os.Getenv("PGDATABASE"),
```

Comment:
Yes, I tried to migrate to pgx all in one go but diff got too big.
So splitting up the changes into smaller bits and will glue it all together when pkg/postgres migrates to pgx.